test/dialogs.cpp

```diff
@@ -26,7 +26,7 @@ static void load_dialog(fs::path path) {
 TEST_CASE("Construct each type of dialog in the engine") try {
 	ResMgr::fonts.pushPath(fs::current_path()/".."/"rsrc"/"fonts");
 	ResMgr::graphics.pushPath(fs::current_path()/".."/"rsrc"/"graphics");
-	fs::path dialogsPath = fs::current_path()/".."/"rsrc"/"dialogs";
+	fs::path dialogsPath = fs::current_path()/".."/".."/"rsrc"/"dialogs";
```

Comment:
This is _very_ suspicious and probably breaks the tests on other targets.

Reply:
yeah, I reverted it. But I have no idea why on some targets, ../rsrc/dialogs doesn't exist when the tests run.

Reply:
Or maybe if `../` relative paths don't work on mac from C++?

Reply:
My guess would be that the current working directory is inconsistent. `fs::current_path()` is equivalent to `pwd`.

Reply:
That wasn't it--`test/SConscript` needed to copy over the files. Fixed now, I think.